kaldb/src/test/java/com/slack/kaldb/chunk/ReadWriteChunkImplTest.java

```diff
@@ -130,43 +130,36 @@ public void testAddAndSearchChunkInTimeRange() {
       assertThat(getCount(REFRESHES_COUNTER, registry)).isEqualTo(1);
       assertThat(getCount(COMMITS_COUNTER, registry)).isEqualTo(1);
 
-      final long expectedStartTimeEpochSecs = messageStartTime / 1000;
-      final long expectedEndTimeEpochSecs = expectedStartTimeEpochSecs + 99;
+      final long expectedEndTimeEpochMs = messageStartTime + (99 * 1000);
```

Comment:
Please rename this variable messageStartTimeMs so the units are more obvious.

Reply:
Done.